Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -622,6 +622,51 @@ IPCCommandResult ES::ExportTitleDone(Context& context, const IOCtlVRequest& requ
 {
   return GetDefaultReply(ExportTitleDone(context));
 }
+
+ReturnCode ES::DeleteSharedContent(const std::array<u8, 20>& sha1) const
+{
+  IOS::ES::SharedContentMap map{Common::FromWhichRoot::FROM_SESSION_ROOT};
+  const std::string content_path = map.GetFilenameFromSHA1(sha1);
+  if (content_path == "unk")
```

Comment:
Maybe a little late to the party (and totally ignorable), but that magic string `"unk"` appears in at least 3 other places. Perhaps its time to make it a constant or something like that?